nixos/modules/services/web-apps/rss-bridge.nix

```diff
@@ -40,27 +60,30 @@ in
 
       user = mkOption {
         type = types.str;
-        default = "nginx";
+        default = cfg.webserver or "rss-bridge";
```

Comment:
i don't think it works like that
the or operator only works on not defined attributes
```
nix-repl> a = {a = null;}
nix-repl> a.a or "hello"
null
nix-repl> a.b or "hello"
"hello"
```

Reply:
good find. did not know that. I reverted this.